kubetest/OWNERS

```diff
@@ -3,6 +3,7 @@
 approvers:
 - BenTheElder
 - krzyzacy
+- shyamjvs
```

Comment:
nit: i'd almost just have Sen in here to keep kubetest sane and drop myself even... this OWNER is because the EKS deployer right? that code should be moved to a subpackage like the kind deployer

Reply:
On second thought: we'll sort that out later. we really need to untangle that main package @krzyzacy